activeactive/skills/skill-restore.sh

```diff
@@ -0,0 +1,88 @@
+#!/bin/bash
+
+
+red=$(tput setaf 1)
+green=$(tput setaf 2)
+yellow=$(tput setaf 3)
+default=$(tput sgr0)
+
+echo "$green Enter backup dir path:"
+read backupDir
+echo "$green Enter skill id:"
+read skillId
+
+uuidArray=(${skillId//-/ })
+uuid=${uuidArray[0]}
+
+echo -e "$green $(date) Backup started \n"
+
+# Validate dependency module
+# $1 - Name of the dependency module
+# $2 - Command to validate module
+function validate_dependency() {
+  eval $2
+  # Next statement is checking last command success
+  if [ $? -ne 0 ]; then
+    echo -e "$red $(date) Please install ******** $1 ***********  ... Exiting $default"
+    exit 1
+  fi
+}
+
+# Validate required modules exits in target setup
+function validate_setup() {
+  validate_dependency jq "jq --version"
+  echo -e "$(date) Successfully validated required dependencies"
+}
+
+
+function create_tenant_secret() {
```

Comment:
Do we need to always create secret? Cant we skip if it already exists?

Reply:
not sure how to check this in script

Reply:
Why are we hitting `ai-deployer/v1/system/namespace/recover` endpoint? To create something. Lets find that and check if it exists.

Reply:
this api will fetch all provisioned tenants and then get secretName and creates the secret

Reply:
i think API handles this, @pramod-rao-uipath  please confirm. I see from ML skill listener you are calling same API

Reply:
done